src/catalog/src/durable.rs

```diff
@@ -120,13 +116,10 @@ pub trait OpenableDurableCatalogState: Debug + Send {
     /// needed.
     ///
     /// `initial_ts` is used as the initial timestamp for new environments.
-    /// `epoch_lower_bound` is used as a lower bound for the epoch that is used by the returned
-    /// catalog.
```

Comment:
This was used during the migration from the stash to persist, but currently is always `None`.